### PR TITLE
Update docs for Manager APIs: /req and /conn/getNode

### PR DESCRIPTION
**Pull request by BigOokie**
Updates to Manager API documentation. Added usage and examples for the `/req` and `/conn/getNode` APIs.

@gz-c please review and let me know if any changes are needed.

### CODE REVIEW

File: docs/api/ManagerAPI.md [hunk 1]
```diff
@@ -166,18 +166,47 @@ Connection: close
 New password length is 4~20.
 ```
 
-### Manager Request
+### Manager Node Request
+Asks the Manager to perform an API request for one of its connected Nodes. Use the `/conn/getNode` API to obtain the `IP` and `Port` for the Node you wish to call an API for.
+
+You must have logged in to the Manager using `/login` and obtained a valid Token from `/token` before calling this API.
```
Comment (mahansky) [inline docs/api/ManagerAPI.md]: token is not required, is it?

File: docs/api/ManagerAPI.md [hunk 2]
```diff
@@ -337,8 +366,12 @@ Error Response:
...
 ```
 
-### Get Node
-Get detailed information from the manager about the specified Node. The Node key must be passed as a query string to the URI.
+### Get Node Information
+Get detailed information from the Manager about the specified Node. The Node key must be passed as a query string to the URI.
+
+Use the `/conn/getAll` to obtain a list of the connect Node keys, then use one of the returned Node Keys as input to this API request.
+
+Note: You must have already logged into the Manager using the `/login` API, and obtained a valid token from the `/token` API.
```
Comment (mahansky) [inline docs/api/ManagerAPI.md]: same thing with the token

File: docs/api/ManagerAPI.md (hunk 1 above)
Comment (BigOokie) [reply docs/api/ManagerAPI.md]: Will check and confirm.

File: docs/api/ManagerAPI.md (hunk 2 above)
Comment (BigOokie) [reply docs/api/ManagerAPI.md]: Will check and confirm

File: docs/api/ManagerAPI.md (hunk 1 above)
Comment (BigOokie) [reply docs/api/ManagerAPI.md]: Corrected

File: docs/api/ManagerAPI.md (hunk 2 above)
Comment (BigOokie) [reply docs/api/ManagerAPI.md]: Corrected

**Pull request merged**